CSS/Styles.css

```diff
@@ -380,9 +476,186 @@ textarea {
 .socialinks li {
   display: inline-block;
   padding: 0;
+  width: fit-content;
+  height: auto;
 }
 
 .socialinks img {
   height: 50%;
   width: auto;
 }
+
+@media screen and (min-width: 768px) {
+  .nav {
+    box-shadow: 0 0.5px 0 #dddfe6;
+    background-image: none;
+    justify-content: center;
+    align-items: center;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .navlinks {
+    display: flex;
+  }
+
+  .homesection {
+    margin-bottom: 60px;
+    flex-wrap: nowrap;
+    height: 100vh;
+  }
+
+  .homewrp,
+  .aboutwrpr {
+    width: 50%;
+    padding-left: 70px;
+    justify-content: flex-start;
+    align-items: flex-start;
+  }
+
+  .maintitle {
+    width: 100%;
+  }
+
+  .mtitle {
+    font-size: 90px;
+    line-height: 90px;
+    padding: 0;
+  }
+
+  .maintxt {
+    width: 90%;
+  }
+
+  .mtext {
+    margin: 0;
+  }
+
+  .decoimg {
+    display: none;
+  }
+
+  .decoimgdsktp {
+    display: block;
+  }
+
+  .works {
+    padding-top: 120px;
+  }
+
+  .project {
+    width: auto;
+    height: 20%;
+    flex: 1 1 25%;
+  }
+
+  .projtitle {
+    background-image: url(../Images/image_dots.svg);
+    background-repeat: no-repeat;
+    background-position: right top;
+    justify-content: center;
+    align-items: flex-start;
+    flex: 1 0 40%;
+  }
+
+  .projtitle h1 {
+    position: relative;
+    transform: rotate(-90deg);
+    width: 100%;
+    text-align: left;
+  }
+
+  .projectwrp {
+    display: none;
+  }
+
+  .mproj {
+    width: auto;
+    flex: 1 1 50%;
+  }
+
+  .mproj .projectwrp {
+    display: block;
+  }
+
+  .shape1 {
+    display: block;
+  }
+
+  .shape2 {
+    display: block;
+  }
+
+  .about {
+    width: 100%;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: nowrap;
+    flex-direction: row-reverse;
+  }
```

Comment:
Your `About` section doesn't quite look like the one from the template design.

Here's your design:

<img width="1440" alt="Screen Shot 2022-06-03 at 13 14 28" src="https://user-images.githubusercontent.com/82426682/171922546-7a113506-6145-4625-b867-a47ba6d9ad40.png">

Here's the template design:

<img width="544" alt="Screen Shot 2022-06-03 at 13 14 58" src="https://user-images.githubusercontent.com/82426682/171922610-1e3a367d-f882-4846-95c6-56d07c52b6e5.png">

Kindly fix it.

Additionally, please fix the margin with the section above, it is significantly narrower than the one from the template design.